client/blocks/plugins-update-manager/schedule-list-cards.tsx

```diff
@@ -57,7 +58,13 @@ export const ScheduleListCards = ( props: Props ) => {
 
 					<div className="schedule-list--card-label">
 						<label htmlFor="last-update">Last Update</label>
-						<span id="last-update"></span>
+						<span id="last-update">
+							{ schedule.last_run_status && (
+								<Badge type={ schedule.last_run_status === 'success' ? 'success' : 'failed' } />
+							) }
+							{ schedule.last_run_timestamp &&
+								moment( schedule.last_run_timestamp * 1000 ).format( MOMENT_TIME_FORMAT ) }
+						</span>
 					</div>
```

Comment:
I would cover a case when the last update is not available.
For example, show the dash character "-."

It will be better than an empty cell